frontend/src/pages/AccountSetup.vue

```diff
@@ -114,55 +122,84 @@
 
 <script lang="ts">
 import { QBtn } from 'quasar';
-import { defineComponent, ref } from '@vue/composition-api';
-import { TransactionResponse } from '@ethersproject/providers';
+import { defineComponent, onMounted, ref } from '@vue/composition-api';
 import BaseButton from 'src/components/BaseButton.vue';
 import useWalletStore from 'src/store/wallet';
 import useAlerts from 'src/utils/alerts';
 import ConnectWalletCard from 'components/ConnectWalletCard.vue';
 
 function useKeys() {
-  const { domainService, getPrivateKeys, userAddress, userEns, spendingKeyPair, viewingKeyPair } = useWalletStore();
+  const {
+    domainService,
+    getPrivateKeys,
+    userAddress,
+    userEns,
+    userCns,
+    spendingKeyPair,
+    viewingKeyPair,
+  } = useWalletStore();
   const { notifyUser, txNotify } = useAlerts();
+  const selectedName = ref<string>(); // ENS or CNS name to be configured
   const carouselBtnRight = ref<QBtn>();
   const keyStatus = ref<'waiting' | 'success' | 'denied'>('waiting');
   const carouselStep = ref('1');
-  const isWaitingForUser = ref(false);
+  const isWaiting = ref(false);
+
+  onMounted(() => checkAndSetName());
+
+  function checkAndSetName() {
+    if (selectedName.value) return;
+    if (userEns.value && !userCns.value) selectedName.value = userEns.value;
+    else if (!userEns.value && userCns.value) selectedName.value = userCns.value;
```

Comment:
I find this segment really hard to read as formatted. I'd strongly favor expanding this out to add braces and newlines. My personal opinion is the readability outweighs the terseness, and also reduces likelihood of an error if the logic is expand at some point in the future.

Reply:
Heh, I do think there is value in terseness because one you grok that line, you benefit from fitting more on screen > can fit more context in my head at once, which I like. But, added braces and comments in 1723291!